tests/MySqlConnector.Tests/CachedProcedureTests.cs

```diff
@@ -198,10 +198,16 @@ param4 INTEGER(3)
 		[InlineData("BINARY(16)", "BINARY", false, 16)]
 		[InlineData("CHAR(36)", "CHAR", false, 36)]
 		[InlineData("ENUM('a','b','c')", "ENUM", false, 0)]
+		[InlineData("NULL", "NULL", false, 0)]
+		[InlineData("TEXT CHARACTER SET utf8 COLLATE utf8_general_ci", "TEXT", false, 0)]
```

Comment:
Should a test for `INT NULL` (or `INT(11) NULL`; what does SingleStore use?) be added here?

Reply:
I added a generic NULL test rather than specific ones (see line [209 below](https://github.com/mysql-net/MySqlConnector/pull/968/files/97e027d41eccce58f520fe554bc1a1fab370a2b7#diff-9c881340147b53d23acf96f2cf99ae82c39cd8cb48760545d8e44bd5cbe0dfe0R209))

SingleStore will return `INT(11) NULL` so the third test on line 186 will test it.